### DISCUSSION TITLE
SSL Error Windows

### DISCUSSION THREAD

**Discussion by jkj787** [kubilus1/autoortho#504] [General]
Im getting an SSL certificate error when opening AO. The message is:

WARNING:downloader:Couldn't update release info <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)>.
I seen this issue has been solved for Linux not sure if it applies to Windows. I am using Windows 10
I have unintalled/reinstalled Dokan,WinFsp, and Python but nothing works. I am unable to download any scenery due to this.
Any help would be much appreciated

**Comment by deep-wb** [comment]
It seems like you're encountering an SSL certificate verification error while trying to access a resource or service related to AO
The error specifically mentions a failure to update release information due to an SSL certificate that has expired.

I recommend [checking the SSL certificate](https://www.ssl2buy.com/wiki/ssl-installation-checker) expiry date, and system date configuration. This error occurs when the valid SSL is not available for the service/domain that you are communicating. If it requires the SSL Certificate, then you/the client may need to acquire and update on the domain/server
